src/Structs/Playlist.js

```diff
@@ -0,0 +1,22 @@
+import Track from './Track';
+
+export default class Playlist {
+  static fromPlaylistObject = (id, playlistObject) => {
+    const playlist = new Playlist(id, playlistObject.Oh.replace(/ playlist$/g, ''));
+    playlistObject.items.map((track) => Track.fromTrackArray(track.Pf.Lc, track.index)).forEach((track) => {
+      playlist.addTrack(track);
+    });
+    return playlist;
+  };
+
+  constructor(id, name) {
+    this.id = id;
+    this.name = name;
+    this.tracks = [];
+  }
+
+  addTrack(track) {
+    this.tracks.push(track);
+    this.tracks.sort((t1, t2) => t1.index - t2.index);
```

Comment:
This isn't too efficient for the initial construction. For `n` playlists, we will be sorting `n` times when we really only need to sort 1 time. Maybe we should add an `addTracks` method which both call a `_sortTracks()` method